scripts/common.py

```diff
@@ -202,6 +202,15 @@ def add_args(parser: argparse.ArgumentParser) -> None:
 config = Config()
 
 
+def get_host_triplet() -> str:
+    if on_linux():
+        return "x86_64-unknown-linux-gnu"
+    elif on_mac():
+        return "x86_64-apple-darwin"
+    else:
+        assert False, "not implemented"
```

Comment:
```suggestion
        raise NotImplemented
```

Reply:
This is just code motion; semantic changes are out of scope.